integration_test/run_adapter_and_wait.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 # apiStage project location dataset store adapter_port finish_port
-gradle run -Dexec.args='--dimse_aet=IMPORTADAPTER --dimse_port='$6' --dicomweb_addr=https://healthcare.googleapis.com/'$1'/projects/'$2'/locations/'$3'/datasets/'$4'/dicomStores/'$5'/dicomWeb --dicomweb_stow_path=/studies --verbose' &
+gradle run -Dexec.args='--dimse_aet=IMPORTADAPTER --dimse_port='$6' --dicomweb_addr=https://healthcare.googleapis.com/'$1'/projects/'$2'/locations/'$3'/datasets/'$4'/dicomStores/'$5'/dicomWeb --dicomweb_stow_path=studies --verbose' &
```

Comment:
We don't want to break existing clients which specify the path starting with "/". Let's change the code of the dicomweb client stowrs method to handle all cases (base url has trailing /, path has a / at the beginning, etc). We can do this by trimming them off of both the base url and path if they exist.

Reply:
Do we actually need dicomweb_stow_path parameter at all? I mean it's present as legacy, but I'm not sure why it was needed in the 1st place. It's always 'studies' anyway (with or without \).

The only other option in stow-rs standard is '/studies/{study}', and difference is that this produces errors on mismatch between url study uid and instance study uid ( http://dicom.nema.org/dicom/2013/output/chtml/part18/sect_6.6.html#sect_6.6.1.1 ). Does not seem particularly important.

Reply:
It's a good point. Ideally we should just have one dicomweb address parameter which goes up to /dicomWeb, which is used by CFIND, CMOVE, and CSTORE.

It's important to keep backwards compatibility here for now. I spoke to Dusan about it and I think the following plan makes sense:
1. (this PR) Support existing clients which specify both paths separately, and handle all cases of where the "/" character is present as I mentioned in the first comment. The reason we can't get rid of the stow path param and hardcode to /studies is that current clients don't specify the full dicom web address in the dicom web address flag. From our readme:
- "--dicomweb_addr=https://healthcare.googleapis.com/v1beta1"
            - "--dicomweb_stow_path=/projects/myproject/locations/us-central1/datasets/mydataset/dicomStores/mydicomstore/dicomWeb/studies"
Note the stow_path is not just /studies here.

2. (follow up PR) Add a new flag for dicom web address which is mandatory to use CFIND and CMOVE, which is the dicomweb path up to /dicomWeb. We can use only this for CFIND and CMOVE. For CSTORE since we have legacy clients we should support both ways of specifying the path for a while. Note that CFIND and CMOVE won't work with the current way of specifying the path in the readme file above, because the complete path is not part of the dicomweb_addr param.

3. (same follow up PR) Mark the old stow path flag as deprecated in a comment, and explain how to migrate to the new way of specifying the flag in the readme, as well as updating the example in the readme.

4. Eventually we can remove the stow path. 

Does this sound good to you?

Reply:
Ok, makes sense. Added  "/" trimming.